library/std/src/fs/tests.rs

```diff
@@ -644,9 +644,9 @@ fn recursive_rmdir_toctou() {
     // Test for time-of-check to time-of-use issues.
     //
     // Scenario:
-    // The attacker wants to get directory contents deleted, to which he does not have access.
-    // He has a way to get a privileged Rust binary call `std::fs::remove_dir_all()` on a
-    // directory he controls, e.g. in his home directory.
+    // The attacker wants to get directory contents deleted, to which they does not have access.
+    // They has a way to get a privileged Rust binary call `std::fs::remove_dir_all()` on a
+    // directory they controls, e.g. in their home directory.
```

Comment:
Unfortunately, "contribution" via regex, without human oversight, is not a good strategy. Pronouns are not context-free. It is "they do not have". I think this is the only legitimate example of something we might want to change, and it is still wrong and thus cannot be merged.